multinet/types.py

```diff
@@ -5,16 +5,15 @@
 EdgeDirection = Literal["all", "incoming", "outgoing"]
 TableType = Literal["all", "node", "edge"]
 
-WorkspacePermissions = TypedDict(
-    "WorkspacePermissions",
-    {
-        "owner": str,  # this is the `sub` property of the `User` dataclass
-        "maintainers": List[str],
-        "writers": List[str],
-        "readers": List[str],
-        "public": bool,
-    },
-)
+
+class WorkspacePermissions(TypedDict):
```

Comment:
Why still inherit from `TypedDict`? I thought you were on a mission to exterminate them?

Reply:
I am still on a mission, I just changed the declaration of `WorkspacePermissions` to use the class based syntax, as it's more clear.